.drone/drone.yml

```diff
@@ -41,6 +41,11 @@ steps:
   - test
   - lint
 
+- name: shellcheck
+  image: koalaman/shellcheck-alpine:stable
+  commands:
+  - apk add make bash && make lint-scripts
```

Comment:
I've follow the pattern `make lint-jsonnet` and `make lint-markdown`. However, it would be easier to just have a shellcheck script file and avoid installing make and bash here.